ai/integrations/autogen/src/unitycatalog/ai/autogen/toolkit.py

```diff
@@ -172,6 +173,7 @@ def func(**kwargs: Any) -> str:
             result = client.execute_function(
                 function_name=function_name,
                 parameters=args_json,
+                autologging_enabled=autologging_is_enabled("anthropic"),
```

Comment:
Perhaps you copy/pasted and forgot to update for each integration? All say "anthropic," but likely should reflect the name of each framework.